utils/locale.go

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2018 DeineAgentur UG https://www.deineagentur.com. All rights reserved.
+ * Licensed under the MIT License. See LICENSE file in the project root for full license information.
+ */
+
+package utils
```

Comment:
No need for `utils` package. All this can stay at the main `gotext` package as it's relevant to its functionality. 

Also see https://blog.golang.org/package-names for reasons to not have a `utils` package.